### ISSUE TITLE
Precision metric for classification

### ISSUE THREAD

**Issue by lars-reimann** [Safe-DS/Stdlib#185]
### Is your feature request related to a problem?

Computing the precision of a classifier for a specific class is a common task.

### Desired solution

* New method called `precision` in `Classifier`
* Required parameter `positive_class` to specify which value should be treated as positive (all other classes are treated as negative)
* Call [`sklearn.metrics.precision`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_score.html#sklearn.metrics.precision_score)
